card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
```

Comment:
Ideally, yes.  The main reason I still need wagn for the engine deck case is because I want the rake tasks.  Should we copy the rake tasks into the new namespace and have two for a while?  Start using decko:task now?  I don't think we need the wagn command for this case as long as the tasks are available.

One of the important things on this branch is getting the generators to load in rails as they should.